### PR TITLE
Fix/cotter enum unpkg

### PR DESCRIPTION
**Pull request by putrikarunia**
Set CotterEnum as static attribute of Cotter

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cotter",
-  "version": "0.3.24",
+  "version": "0.3.25-beta.0",
```
Comment (albertpurnama) [inline package.json:3]: still beta?

File: src/indexumd.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import Cotter from "./Cotter";
 import CotterEnum from "./enum";
 
 class CotterExport extends Cotter {
-  CotterEnum: CotterEnum;
+  static CotterEnum = CotterEnum;
```
Comment (albertpurnama) [inline src/indexumd.ts:5]: I don't understand, why not export `CotterEnum` on line 8 ? just export both `CotterEnum` and `CotterExport`

File: package.json (hunk 1 above)
Comment (putrikarunia) [reply package.json:3]: Yea I'll update to 0.3.25 on master

File: src/indexumd.ts (hunk 2 above)
Comment (putrikarunia) [reply src/indexumd.ts:5]: Becos I think can only export 1 thing. So in the webpack.config.js, can define what is the export name. I set the name as `Cotter` that's why `CotterExport` is accessible as `Cotter` when using the `<script>` tag.

**Review by albertpurnama — APPROVED** [review]

**Pull request merged**